modules/_labs/select/react/lib/scrolling.ts

```diff
@@ -0,0 +1,59 @@
+const clientHeight = (elem: HTMLElement): number => {
+  const computedStyle = window.getComputedStyle(elem, null);
+  const borderTopWidth = parseInt(computedStyle.getPropertyValue('border-top-width'), 10);
+  const borderBottomWidth = parseInt(computedStyle.getPropertyValue('border-bottom-width'), 10);
+  return elem.offsetHeight - borderTopWidth - borderBottomWidth;
+};
+
+// Modified from https://gist.github.com/hsablonniere/2581101
+export const scrollIntoViewIfNeeded = (elem: HTMLElement, centerIfNeeded = true): void => {
+  const parent: HTMLElement | null = elem.parentElement;
```

Comment:
You generally shouldn't provide a type to a `const` and let it infer from the assignment. You could inadvertently widen the type:

```suggestion
  const parent = elem.parentElement;
```

Reply:
Fixed.